features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1888,6 +1916,10 @@ CREATE INDEX IDX_IDN_CERTIFICATE_UUID_TID ON IDN_CERTIFICATE (UUID, TENANT_ID);
 CREATE INDEX IDX_IDN_RULE_UUID_TID ON IDN_RULE (UUID, TENANT_ID);
 CREATE INDEX IDX_IDN_RULE_REF_RID_TID ON IDN_RULE_REFERENCES (RULE_ID, TENANT_ID);
 
+-- WEBHOOKS --
+CREATE INDEX IDX_IDN_WEBHOOK_UUID_TID ON IDN_WEBHOOK (UUID, TENANT_ID);
+CREATE INDEX IDX_IDN_WEBHOOK_EVENTS_WID ON IDN_WEBHOOK_EVENTS (WEBHOOK_ID);
```

Comment:
I don't think this index is needed. For the primary key and index is default created